articles/article-105-livelock.md

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Oops! My computer keeps freezing, could it be a livelock?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Yes, It could be. A livelock is one of the reasons that might freeze your system. But, what is a livelock?  
+
+Suppose two gentlemen sitting in a restaurant and the waiter brings the order without enough cutlery. The first one has a fork and the second one has a knife. To be able to eat the first asks the second for the knife and at the same time the second asks the first for the fork. They exchange the cutlery, and the situation happens again; the first has the knife and the second has the fork. They keep exchanging the cutlery without eating. 
```

Comment:
Language: Should there be a comma after To be able to eat?

Reply:
Language: To improve clarity, one could write: The first gentlement asks the second for the knife

Reply:
Yes, I have to add a comma after the "To be able to eat". 
ok, it is better, using "gentleman" instead of one.